src/memote/checks/Consistency.jl

```diff
@@ -129,4 +129,56 @@ function model_is_consistent(
     termination_status(opt_model) == OPTIMAL
 end
 
+"""
+$(TYPEDSIGNATURES)
+
+Returns a list of all metabolites that aren't part of any reactions.
+"""
+function find_disconnected_metabolites(model::MetabolicModel)
+    mids = deepcopy(metabolites(model))
+    for rid in reactions(model)
+        rid_m = reaction_stoichiometry(model, rid)
+        for mid in mids
+            haskey(rid_m, mid) && deleteat!(mids, findall(x->x == mid, mids))
+        end
+    end
+    return mids
+end
+
+"""
+$(TYPEDSIGNATURES)
+
+A function to find reactions able to carry unlimited flux under default conditions.
+The function compares the fluxes of the model calculated by FVA (using flux_variability_analysis_dict)
+with the median bounds of the model (+/- treshold) and returns the fluxes which are grearter or smaller than 
+the bounds in a two seperate dictionaries.
+"""
+function unbounded_flux_in_default_medium(model::MetabolicModel, fva_result::Tuple{Any,Any}, config = memote_config)
+    tol = config.consistency.tolerance_threshold
+
+    if isempty(fva_result)
+        throw(ArgumentError("fva_result is empty"))
+    end
+
+    low_unlimited_flux = Dict{String, Vector{Any}}()
+    high_unlimited_flux = Dict{String, Vector{Any}}()
+
+    min_fluxes, max_fluxes = fva_result
+    lower_bound, upper_bound = median_bounds(model)
+
+    for rid in reactions(model), rid2 in reactions(model)
```

Comment:
nice! isn't Julia a pleasure to use?

Reply:
actually it might even be better to avoid indexing and just walk along the structure of `min_fluxes`:

```suggestion
    for (rid,d) in min_fluxes for (frid, min_flux) in d
```

and the rest of the code than goes a bit easier (and with less lookups):
```julia
        if min_flux < lower_bound || isapprox(min_flux, lower_bound; atol = tol)
            low_unlimited_flux[rid] = [frid, min_flux]   --btw this should be a tuple, not sure if the types would work nicely otherwise
        end

Reply:
beautiful :)